src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/NoRemissionHWFCallbackHandler.java

```diff
@@ -40,8 +46,21 @@ public List<CaseEvent> handledEvents() {
     }
 
     private CallbackResponse noRemissionHWF(CallbackParams callbackParams) {
+        CaseData caseData = callbackParams.getCaseData();
+        CaseData.CaseDataBuilder<?, ?> updatedData = caseData.toBuilder()
+            .businessProcess(BusinessProcess.ready(NOTIFY_LIP_CLAIMANT_HWF_OUTCOME));
+
```

Comment:
this has to be done in all the HWF handlers. you can create a common util method for this.